corehq/apps/builds/templates/builds/bootstrap5/edit_menu.html

```diff
@@ -15,103 +15,89 @@
-        </tr>
-        </thead>
-        <tbody data-bind="foreach: versions">
-        <tr>
-          <td>
-            <select class="form-control" data-bind="
-                                    options: $root.available_versions,
-                                    value: version
-                                ">
-            </select>
-          </td>
-          <td>
-            <input class="form-control" type="text" data-bind="value: badge">
-          </td>
-          <td>
-            <input type="checkbox" value="superuser-only" data-bind="checked: superuser_only"/>
-          </td>
-          <td>
-            <button type="button" class="btn btn-outline-danger" data-bind="click: $root.removeVersion">
-              <i class="fa fa-remove"></i>
-            </button>
-          </td>
-        </tr>
-        </tbody>
-        <tr>
-          <td colspan="5">
-            <button data-bind="click: addVersion" class="btn btn-outline-primary">
-              <i class="fa fa-plus"></i>
-              {% trans "Add a Version" %}
-            </button>
-          </td>
-        </tr>
-      </table>
+    <div class="card-text">
+      <form id="menu-form" method="POST">{% csrf_token %}
+        <table id="menu-table" class="table table-striped">
+          <thead>
+          <tr>
+            <th>{% trans "Version" %}</th>
+            <th>{% trans "Label" %}</th>
+            <th>{% trans "Superuser Only" %}</th>
+            <th></th>
+          </tr>
+          </thead>
+          <tbody data-bind="foreach: versions">
+          <tr>
+            <td>
+              <select class="form-control" data-bind="
+                                      options: $root.available_versions,
+                                      value: version
+                                  ">
+              </select>
+            </td>
+            <td>
+              <input class="form-control" type="text" data-bind="value: label">
+            </td>
+            <td>
+              <input type="checkbox" value="superuser-only" data-bind="checked: superuser_only"/>
+            </td>
+            <td>
+              <button type="button" class="btn btn-outline-danger" data-bind="click: $root.removeVersion">
+                <i class="fa fa-remove"></i>
+              </button>
+            </td>
+          </tr>
+          </tbody>
+        </table>
+        <button data-bind="click: addVersion" class="btn btn-outline-primary">
+          <i class="fa fa-plus"></i>
+          {% trans "Add a Version" %}
+        </button>
 
-      <hr/>
+        <hr/>
 
-      <fieldset>
-        <legend>Defaults</legend>
-        <div class="form-group">
-          <label class="form-label col-md-3" for="select-default-1">
-            {% trans "Default 1.x Build" %}
-          </label>
-          <div class="col-md-8">
+        <fieldset>
+          <legend>Defaults</legend>
+          <div class="mb-3">
+            <label class="form-label" for="select-default-1">
```

Comment:
and `col-form-label` on here. I think those two things would display the label next to the field on larger screens.